### PR TITLE
Uitvoeren schriftelijke test & Evalueren testresultaat

### PR DESCRIPTION
**Pull request by ernstbolt**

### CODE REVIEW

File: content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+> 
+> In dit geval zijn de **testsoorten** ****unit test****, **integratietest**, **systeemtest**, en **acceptatietest** van toepassing op het testen van de functionaliteit voor het toevoegen van een product aan het winkelwagentje.
+
+## Hoe zit het selecteren van **testsoorten** in elkaar?
+Voordat we ingaan op het selecteren van **testsoorten** bekijken we het model van testen: het **V-model**. Daarna worden de verschillende **testsoorten** beschreven en in relatie gebracht met het **V-model**.
+
+### **V-model**
+Het **V-model** is een model dat het softwareontwikkelproces en het testproces visueel in kaart brengt. Het model toont de relatie tussen de fasen van softwareontwikkeling en de bijbehorende testfasen. Het model heeft een **V-vorm** waarbij de linkerhelft de verificatie-fasen toont en de rechterhelft de validatie-fasen.
+
+![[V-model-en-test-pyramide.png]]
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md:38]: Mist figuur onderschrift

File: content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+- **Verificatie** is het proces van **testen of het product voldoet aan de specificaties en de ontwerpen**. Het controleert of de software volgens de juiste procedures en specificaties is gebouwd.  
+- **Validatie** gaat verder en kijkt of het **eindproduct voldoet aan de wensen en behoeften van de klant**. Het bevestigt of de software daadwerkelijk de juiste oplossing biedt voor de klant.
+
+In het **V-model** is de verificatie gekoppeld aan de ontwerp- en ontwikkelingsfasen aan de linkerkant van de "V", terwijl validatie wordt uitgevoerd in de testfasen aan de rechterkant. Het model benadrukt het belang van het plannen van testfasen parallel aan de ontwikkelingsfasen, zodat het product niet alleen technisch correct is, maar ook daadwerkelijk aan de klantvereisten voldoet.
+
+**V-model, SSDLC en SCRUM**  
+De fasen van het V-model zijn naadloos te combineren met de fasering in de SSDLC en SCRUM. Het is aan het ontwikkelteam om in overleg met de opdrachtgever en stakeholders hiertussen een koppeling te leggen. Dit is een taak die bij het opzetten van het project uitgevoerd moet worden.  
+
+### **Testsoorten**
+De ****testsoorten**** kunnen als volgt worden onderscheiden:
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md:91]: `****testsoorten****` -> `**testsoorten**`

File: content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+*Het verschil tussen **verificatie** en **validatie**.*
+- **Verificatie** is het proces van **testen of het product voldoet aan de specificaties en de ontwerpen**. Het controleert of de software volgens de juiste procedures en specificaties is gebouwd.  
+- **Validatie** gaat verder en kijkt of het **eindproduct voldoet aan de wensen en behoeften van de klant**. Het bevestigt of de software daadwerkelijk de juiste oplossing biedt voor de klant.
+
+In het **V-model** is de verificatie gekoppeld aan de ontwerp- en ontwikkelingsfasen aan de linkerkant van de "V", terwijl validatie wordt uitgevoerd in de testfasen aan de rechterkant. Het model benadrukt het belang van het plannen van testfasen parallel aan de ontwikkelingsfasen, zodat het product niet alleen technisch correct is, maar ook daadwerkelijk aan de klantvereisten voldoet.
+
+**V-model, SSDLC en SCRUM**  
+De fasen van het V-model zijn naadloos te combineren met de fasering in de SSDLC en SCRUM. Het is aan het ontwikkelteam om in overleg met de opdrachtgever en stakeholders hiertussen een koppeling te leggen. Dit is een taak die bij het opzetten van het project uitgevoerd moet worden.  
+
+### **Testsoorten**
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md:90]: Deze OI heb je heb de casus ook al staan, is nu erg dubbel. Uit de casus weghalen lijkt me beste optie

File: content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+- **Systeemtest**:  
+  **Systeemtesten** zijn gericht op het testen van het gehele systeem in een realistische omgeving. Hierbij wordt gecontroleerd of de applicatie als geheel goed functioneert en voldoet aan de gestelde eisen. Het doel is om te testen of alle onderdelen van het systeem naadloos samenwerken en of het systeem werkt zoals verwacht in zijn volledige context.
+
+- **Acceptatietest**:  
+  Bij **acceptatietests** wordt het systeem getest vanuit het perspectief van de eindgebruiker. Het doel is om te controleren of de applicatie voldoet aan de bedrijfsvereisten en verwachtingen van de klant. Deze tests worden vaak uitgevoerd door de klant zelf of door vertegenwoordigers van de klant en zijn bedoeld om te valideren of het systeem klaar is voor productie.
+
+Elke **testsoort** heeft zijn eigen focus en rol binnen het ontwikkelings- en testproces. Unittests worden vaak in de vroege ontwikkelingsfasen uitgevoerd, terwijl **integratietests, systeemtests en acceptatietests** later in het proces worden uitgevoerd om de kwaliteit van het gehele systeem te waarborgen.
+
+### Kiezen van **testsoorten**
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md:107]: ** in de titel

File: content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+De testinspanning verwijst naar de afspraak die is gemaakt over welke **testsoorten** worden toegepast en de hoeveelheid analyse die wordt uitgevoerd voor een specifieke use case. Het is de inschatting van de benodigde middelen en tijd voor het uitvoeren van verschillende **testsoorten**, afhankelijk van de complexiteit van de use case en de bijbehorende risico's. Dus de testinspanning bepaalt welke **testsoorten** worden toegepast op een use case.
+
+Voor elk type **testsoort** zijn meerdere **testvormen** te kiezen. Een overzicht en beschrijving hiervan is te vinden in het [TMAP - Overzicht **testvormen**.pdf|overzicht **testvormen** van TMap].
+
+## Hoe gebruik je het verzamelen van **testsoorten**?
+Het kiezen van ****testsoorten**** is van belang zodra bepaald is welke testcases voor een testscenario getest moeten gaan worden. Afhankelijk van de afgesproken testinspanning wordt bepaald welke **testsoorten** daadwerkelijk worden toegepast. 
+
+---
+
+> Volgende stap: [5. Uitleg opstellen schriftelijke testcase|Uitleg opstellen schriftelijke testcase]
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md:117]: Bestand verwijzingen moeten met twee `[[`

File: content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+### Kiezen van **testsoorten**
+De testinspanning verwijst naar de afspraak die is gemaakt over welke **testsoorten** worden toegepast en de hoeveelheid analyse die wordt uitgevoerd voor een specifieke use case. Het is de inschatting van de benodigde middelen en tijd voor het uitvoeren van verschillende **testsoorten**, afhankelijk van de complexiteit van de use case en de bijbehorende risico's. Dus de testinspanning bepaalt welke **testsoorten** worden toegepast op een use case.
+
+Voor elk type **testsoort** zijn meerdere **testvormen** te kiezen. Een overzicht en beschrijving hiervan is te vinden in het [TMAP - Overzicht **testvormen**.pdf|overzicht **testvormen** van TMap].
+
+## Hoe gebruik je het verzamelen van **testsoorten**?
+Het kiezen van ****testsoorten**** is van belang zodra bepaald is welke testcases voor een testscenario getest moeten gaan worden. Afhankelijk van de afgesproken testinspanning wordt bepaald welke **testsoorten** daadwerkelijk worden toegepast.
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md:113]: **testsoorten**

File: content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+
+Elke **testsoort** heeft zijn eigen focus en rol binnen het ontwikkelings- en testproces. Unittests worden vaak in de vroege ontwikkelingsfasen uitgevoerd, terwijl **integratietests, systeemtests en acceptatietests** later in het proces worden uitgevoerd om de kwaliteit van het gehele systeem te waarborgen.
+
+### Kiezen van **testsoorten**
+De testinspanning verwijst naar de afspraak die is gemaakt over welke **testsoorten** worden toegepast en de hoeveelheid analyse die wordt uitgevoerd voor een specifieke use case. Het is de inschatting van de benodigde middelen en tijd voor het uitvoeren van verschillende **testsoorten**, afhankelijk van de complexiteit van de use case en de bijbehorende risico's. Dus de testinspanning bepaalt welke **testsoorten** worden toegepast op een use case.
+
+Voor elk type **testsoort** zijn meerdere **testvormen** te kiezen. Een overzicht en beschrijving hiervan is te vinden in het [TMAP - Overzicht **testvormen**.pdf|overzicht **testvormen** van TMap].
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md:110]: PDF verwijzing klopt niet, volgens mij is er zo een conflict met PR #101 aangezien ik hetzelfde daar ook zag staan

File: content/2. Processen/5. Testproces/5. Uitleg opstellen schriftelijke testcase.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
+---
+title: 5. Uitleg Opstellen schriftelijke testcase 
+taxonomie:
+  - to-2.2.OpstellenSelecteren-**testsoort**.OI
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/5. Uitleg opstellen schriftelijke testcase.md:4]: `**testsoort**` -> `testsoort`

File: content/2. Processen/5. Testproces/5. Uitleg opstellen schriftelijke testcase.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
+---
+title: 5. Uitleg Opstellen schriftelijke testcase
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/5. Uitleg opstellen schriftelijke testcase.md:2]: `Opstellen` -> `opstellen`

File: content/2. Processen/5. Testproces/5. Uitleg opstellen schriftelijke testcase.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+>
+> Vervolgens beschrijft Lisa de **teststappen** die uitgevoerd moeten worden. De gebruiker moet eerst naar de inlogpagina navigeren, vervolgens een geldig e-mailadres en wachtwoord invoeren en ten slotte op de knop **"Inloggen"** klikken. Ze zorgt ervoor dat de instructies helder en eenduidig zijn, zodat elke tester dezelfde werkwijze volgt.  
+> 
+> Tot slot definieert Lisa de **postconditie**: na succesvolle invoer van de inloggegevens moet de gebruiker worden doorgestuurd naar de dashboardpagina. Dit helpt om te bepalen of de test geslaagd is of niet. Door het ontwerp van deze testcase zorgvuldig uit te werken, legt Lisa de basis voor een effectieve testuitvoering en een betrouwbare evaluatie van de inlogfunctionaliteit.
+> Hier is het resultaat dat Lisa hee**ft vastgelegd in het **testontwerp**.
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/5. Uitleg opstellen schriftelijke testcase.md:23]: Witregel erboven

Comment (martijnschuman) [inline content/2. Processen/5. Testproces/5. Uitleg opstellen schriftelijke testcase.md:23]: `hee**ft` -> `heeft`

File: content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+- Een indicatie of de test is geslaagd of mislukt  
+
+Bij een mislukte test wordt de afwijking vastgelegd als een bevinding, zodat deze later geanalyseerd en opgelost kan worden.  
+
+**Registreren van bevindingen**  
+Een bevinding (bug of defect) wordt geregistreerd in een testrapportage of een speciaal bevindingensysteem. Dit rapport bevat de volgende elementen:  
+
+- Een duidelijke omschrijving van het probleem  
+- De omstandigheden waarin de fout optreedt  
+- De ernst van de bevinding (kritiek, hoog, gemiddeld, laag)
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md:52]: Misschien een idee om een tip block toe te voegen over hoe je dit kan bepalen. Wanneer is hij laag en wanneer gemiddeld? etc

File: content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+
+## Hoe gebruik je het uitvoeren van een handmatige test?
+Een **handmatige test** wordt uitgevoerd in verschillende situaties. Wanneer een **nieuwe functionaliteit** is ontwikkeld en nog niet eerder getest is, is een **handmatige test** belangrijk om te controleren of deze voldoet aan de gestelde eisen en specificaties. Daarnaast wordt handmatig testen vaak ingezet bij **regressietesten**, waarbij wordt gecontroleerd of bestaande functionaliteiten nog correct werken na wijzigingen in de codebase. Dit is vooral van belang wanneer er geen volledige geautomatiseerde testset beschikbaar is.  
+
+Ook bij het **verifiëren van bugfixes** speelt handmatig testen een belangrijke rol. Nadat een defect is opgelost, wordt gecontroleerd of het probleem daadwerkelijk verholpen is. Dit gebeurt doorgaans als onderdeel van een hertest of retest. Daarnaast wordt **handmatige test**ing toegepast bij **usability testing**, waarbij de gebruikerservaring wordt beoordeeld. Hierbij kijkt men of de applicatie intuïtief en gebruiksvriendelijk is.
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md:75]: `**handmatige test**ing`

File: content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+
+Ook bij het **verifiëren van bugfixes** speelt handmatig testen een belangrijke rol. Nadat een defect is opgelost, wordt gecontroleerd of het probleem daadwerkelijk verholpen is. Dit gebeurt doorgaans als onderdeel van een hertest of retest. Daarnaast wordt **handmatige test**ing toegepast bij **usability testing**, waarbij de gebruikerservaring wordt beoordeeld. Hierbij kijkt men of de applicatie intuïtief en gebruiksvriendelijk is.  
+
+Soms worden **handmatige test**s **ad-hoc** of exploratief uitgevoerd, zonder vooraf gedefinieerde testcases, om onverwachte problemen te ontdekken. Dit kan nuttig zijn voor het identificeren van edge-cases die anders over het hoofd worden gezien. In gevallen waarin automatisering niet haalbaar of kostbaar is, zoals bij visuele validaties of tests die menselijke interpretatie vereisen, wordt handmatig testen ook ingezet.
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md:77]: `**handmatige test**s`

File: content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+
+Soms worden **handmatige test**s **ad-hoc** of exploratief uitgevoerd, zonder vooraf gedefinieerde testcases, om onverwachte problemen te ontdekken. Dit kan nuttig zijn voor het identificeren van edge-cases die anders over het hoofd worden gezien. In gevallen waarin automatisering niet haalbaar of kostbaar is, zoals bij visuele validaties of tests die menselijke interpretatie vereisen, wordt handmatig testen ook ingezet.  
+
+Verschillende rollen binnen een team kunnen betrokken zijn bij **handmatige test**s. **Testers**, zoals **software test engineers** of **QA engineers**, voeren deze tests uit op basis van testmethodologieën en tools. **Ontwikkelaars** testen soms ook handmatig, vooral tijdens unit- en integratietests voordat zij code overdragen aan testers. Daarnaast kunnen business analisten of product owners **handmatige test**s uitvoeren om te verifiëren of de software voldoet aan zakelijke vereisten. Tot slot worden **handmatige test**s vaak uitgevoerd door eindgebruikers tijdens de User Acceptance Testing (UAT), waarbij de software in een realistische omgeving wordt getest om te bepalen of deze geschikt is voor productie.
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md:79]: `**handmatige test**s`

File: content/2. Processen/5. Testproces/7. Uitleg evalueren testresultaat.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+
+> [!TIP] Casus  
+> Lisa liep de laatste testresultaten nog eens door. Veel testen waren opnieuw mislukt, en hoewel sommige bugs al bij de vorige testcyclus waren gerapporteerd, stonden ze nog steeds open. Ze wist dat niet alle fouten direct konden worden opgelost, maar de deadline naderde snel.  
+> 
+> Met een notitieblok vol bevindingen liep ze naar de werkruimte van Thomas, de lead developer. “We hebben een probleem,” begon ze. “Er zijn nog vijf kritieke bugs die de werking van het systeem beïnvloeden. Vooral de betalingsmodule blijft vastlopen.” Thomas fronste en schoof zijn toetsenbord opzij. “Welke zijn echt blokkerend?” vroeg hij. Lisa wees naar haar scherm. “Deze hier. Zonder oplossing kunnen gebruikers hun bestelling niet afronden.”  
+> 
+> Samen beoordeelden ze welke defecten prioriteit moesten krijgen. Sommige waren relatief eenvoudig te verhelpen, maar anderen vergden diepere aanpassingen in de code. Na hun gesprek stapten ze naar de projectleider, Mark, die de impact op de opleverdatum moest inschatten.
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/7. Uitleg evalueren testresultaat.md:15]: Mark was in de vorige OI casus nog de ontwikkelaar. In de tusssentijd heeft hij dus een promotie gehad, goed bezig Mark! 
Of dit is een andere Mark 🤔🤔

---

Andere naam voor Mark gebruiken

File: content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md [hunk 16]
```diff
@@ -0,0 +1,117 @@
...
+
+In het **V-model** is de verificatie gekoppeld aan de ontwerp- en ontwikkelingsfasen aan de linkerkant van de "V", terwijl validatie wordt uitgevoerd in de testfasen aan de rechterkant. Het model benadrukt het belang van het plannen van testfasen parallel aan de ontwikkelingsfasen, zodat het product niet alleen technisch correct is, maar ook daadwerkelijk aan de klantvereisten voldoet.
+
+**V-model, SSDLC en SCRUM**  
+De fasen van het V-model zijn naadloos te combineren met de fasering in de SSDLC en SCRUM. Het is aan het ontwikkelteam om in overleg met de opdrachtgever en stakeholders hiertussen een koppeling te leggen. Dit is een taak die bij het opzetten van het project uitgevoerd moet worden.  
+
+### **Testsoorten**
+De ****testsoorten**** kunnen als volgt worden onderscheiden:
+
+- **Unit test**:
```
Comment (martijnschuman) [inline content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md:93]: Misschien een idee om te benoemen dat unit testen ook perfect in een CI-CD / ontwikkelstraat / github action... pipeline kunnen worden opgenomen

File: content/2. Processen/5. Testproces/7. Uitleg evalueren testresultaat.md [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+> 
+> Samen beoordeelden ze welke defecten prioriteit moesten krijgen. Sommige waren relatief eenvoudig te verhelpen, maar anderen vergden diepere aanpassingen in de code. Na hun gesprek stapten ze naar de projectleider, Mark, die de impact op de opleverdatum moest inschatten.  
+> 
+> “Als we dit nu aanpakken, schuift de release mogelijk met twee dagen op,” zei Thomas. Lisa voegde eraan toe: “We kunnen enkele minder kritieke fouten accepteren als bekend issue, mits er een tijdelijke workaround is.” Mark knikte bedachtzaam. “Laten we dit bespreken met de product owner. Als het betalingsprobleem opgelost is, kunnen we misschien tóch door met de release.”  
+> 
+> Die middag werd de beslissing genomen: een paar kleine bugs werden geaccepteerd als laag risico, maar de kritieke problemen moesten eerst worden opgelost. Lisa noteerde alles in de testrapportage en bereidde zich voor op een nieuwe testcyclus. Het werk was nog niet klaar, maar de software werd niet zomaar vrijgegeven zonder een zorgvuldige evaluatie.
+
+## Hoe zit het evalueren van het testresultaat in elkaar?
```
Comment (TeunvdKleij) [inline content/2. Processen/5. Testproces/7. Uitleg evalueren testresultaat.md:21]:
```suggestion
## Hoe zit het evalueren van testresultaten in elkaar?
```

File: content/2. Processen/5. Testproces/5. Uitleg opstellen schriftelijke testcase.md [hunk 18]
```diff
@@ -0,0 +1,46 @@
...
+
+- **Testscenario**: De testcase wordt gebaseerd op een testscenario dat de bredere context en het doel van de test beschrijft. Het testscenario bepaalt welke aspecten van de functionaliteit worden getest.  
+- **Requirements**: De functionele en niet-functionele eisen vormen de basis voor de testcase. Ze geven aan wat het systeem moet doen en welke criteria moeten worden getest.  
+- **Risicoanalyse**: De mate van testinspanning wordt bepaald op basis van risico’s. Kritieke functionaliteiten vereisen uitgebreidere testcases.  
+- **Teststrategie**: De gekozen testaanpak en methodologie beïnvloeden hoe gedetailleerd de testcase wordt beschreven. Dit kan variëren van gestructureerde testcases tot meer exploratieve testvormen.  
+- **Testomgeving**: De omgeving waarin de test wordt uitgevoerd kan invloed hebben op de testcase, zoals het type apparaat, de browser of de netwerkcondities.  
+- **Testdata**: De beschikbaarheid en juistheid van testdata zijn essentieel. De testcase moet aangeven welke invoerwaarden nodig zijn en hoe deze worden beheerd.  
+
+Al deze factoren samen bepalen hoe een testcase wordt opgesteld en hoe effectief deze bijdraagt aan het testproces.
+
```
Comment (TeunvdKleij) [inline content/2. Processen/5. Testproces/5. Uitleg opstellen schriftelijke testcase.md:43]: Er mist hier nog de laatste vraag van "Hoe gebruik je"

File: content/2. Processen/5. Testproces/7. Uitleg evalueren testresultaat.md (hunk 15 above)
Comment (ernstbolt) [reply content/2. Processen/5. Testproces/7. Uitleg evalueren testresultaat.md:15]: Het is Leo geworden.

File: content/2. Processen/5. Testproces/7. Uitleg evalueren testresultaat.md [hunk 19]
```diff
@@ -0,0 +1,31 @@
...
+> 
+> Samen beoordeelden ze welke defecten prioriteit moesten krijgen. Sommige waren relatief eenvoudig te verhelpen, maar anderen vergden diepere aanpassingen in de code. Na hun gesprek stapten ze naar de projectleider, Mark, die de impact op de opleverdatum moest inschatten.  
+> 
+> “Als we dit nu aanpakken, schuift de release mogelijk met twee dagen op,” zei Thomas. Lisa voegde eraan toe: “We kunnen enkele minder kritieke fouten accepteren als bekend issue, mits er een tijdelijke workaround is.” Mark knikte bedachtzaam. “Laten we dit bespreken met de product owner. Als het betalingsprobleem opgelost is, kunnen we misschien tóch door met de release.”  
+> 
+> Die middag werd de beslissing genomen: een paar kleine bugs werden geaccepteerd als laag risico, maar de kritieke problemen moesten eerst worden opgelost. Lisa noteerde alles in de testrapportage en bereidde zich voor op een nieuwe testcyclus. Het werk was nog niet klaar, maar de software werd niet zomaar vrijgegeven zonder een zorgvuldige evaluatie.
```
[the remark this answers is not in the document]
Comment (ernstbolt) [reply content/2. Processen/5. Testproces/7. Uitleg evalueren testresultaat.md:19]: ; is evil gewone mensentaal.

File: content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md [hunk 20]
```diff
@@ -0,0 +1,81 @@
...
+
+Verschillende rollen binnen een team kunnen betrokken zijn bij **handmatige tests**. **Testers**, zoals **software test engineers** of **QA engineers**, voeren deze tests uit op basis van testmethodologieën en tools. **Ontwikkelaars** testen soms ook handmatig, vooral tijdens unit- en integratietests voordat zij code overdragen aan testers. Daarnaast kunnen business analisten of product owners **handmatige tests** uitvoeren om te verifiëren of de software voldoet aan zakelijke vereisten. Tot slot worden **handmatige test**s vaak uitgevoerd door eindgebruikers tijdens de User Acceptance Testing (UAT), waarbij de software in een realistische omgeving wordt getest om te bepalen of deze geschikt is voor productie.  
+---
+
+> Volgende stap: [7. Uitleg evalueren testresultaat |Uitleg evalueren testresultaat]
```
Comment (TeunvdKleij) [inline content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md:81]: Bestand links moeten met [[, niet [

File: content/2. Processen/5. Testproces/7. Uitleg evalueren testresultaat.md [hunk 21]
```diff
@@ -0,0 +1,31 @@
+---
+title: 7. Uitleg evalueren testresultaat 
+taxonomie: 
+  - to-2.2.Evalueren-testresultaat.OI
+---
+
+## Wat is het evalueren testresultaat?
```
[the remark this answers is not in the document]
Comment (TeunvdKleij) [reply content/2. Processen/5. Testproces/7. Uitleg evalueren testresultaat.md:7]: Dit is nu geen correcte Nederlandse zin. Dit moet of "Wat is het evalueren van een testresultaat?" of "Wat is het evalueren van testresultaten?" zijn.

File: content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md
[the remark this answers is not in the document]
Comment (ernstbolt) [reply content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md:1]: Verwijderd.

File: content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md [hunk 22]
```diff
@@ -0,0 +1,83 @@
...
+  - to-2.2.Uitvoeren-handmatige-test.OI
+---
+
+## Wat is het uitvoeren van een handmatige test?
+Het uitvoeren van een **handmatige test** betekent dat een tester zonder geautomatiseerde tools of scripts een test uitvoert om te controleren of een functionaliteit werkt zoals verwacht.
+
+> [!TIP] Casus
+> Lisa zuchtte diep en staarde naar haar scherm. Voor de derde keer deze week voerde ze de functionele test uit op de nieuwste versie van de applicatie. De test was geen onbekende voor haar; ze kende de stappen uit haar hoofd. Toch doorliep ze ze zorgvuldig, één voor één, om geen detail over het hoofd te zien.  
+> 
+> Het duurde niet lang voordat de eerste test faalde. Lisa noteerde het in haar testdocument: *"Stap 4: invoer wordt niet correct verwerkt. Verwacht resultaat: bevestigingsbericht. Werkelijk resultaat: foutmelding."* Ze ging verder, maar al snel volgden meer rode kruisen in haar testscript. Hoe verder ze kwam, hoe duidelijker het werd: deze versie zat vol problemen.
```
[the remark this answers is not in the document]
Comment (ernstbolt) [reply content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md:13]: Met testscript wordt de beschrijving van de stappen van de test bedoeld, dus de testcase. Ik heb het aangepast.

File: content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md (hunk 11 above)
Comment (ernstbolt) [reply content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md:52]: Toegevoegd. Mooi tip!

File: content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md
Comment (TeunvdKleij) [inline content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md:1]: Het lijkt mij niet de bedoeling dat deze file verwijderd moet worden, of heb ik het fout?

File: content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md [hunk 23]
```diff
@@ -0,0 +1,87 @@
...
+> [!TIP] Classificeren ernst van de bevinding  
+> Het is aan te raden om een concrete indeling te gebruiken voor de ernst van de bevinding. Bijvoorbeeld:  
+> Urgent - blokkerend voor het proces, geen 'work arounds mogelijk'
+> Hoog - blokkerend voor het proces, 'work around' mogelijk
+> Midden - Niet blokkerend voor het proces 
+> Laag - Cosmetisch, 'nice to have'
+
+### Bespreken en oplossen van bevindingen  
+Nadat de testresultaten en bevindingen zijn vastgelegd, worden ze besproken met de ontwikkelaars. Het doel hiervan is:  
+
```
Comment (TeunvdKleij) [inline content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md:65]:
```suggestion
```

File: content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md [hunk 24]
```diff
@@ -0,0 +1,87 @@
...
+## Hoe gebruik je het uitvoeren van een handmatige test?
+Een **handmatige test** wordt uitgevoerd in verschillende situaties. Wanneer een **nieuwe functionaliteit** is ontwikkeld en nog niet eerder getest is, is een **handmatige test** belangrijk om te controleren of deze voldoet aan de gestelde eisen en specificaties. Daarnaast wordt handmatig testen vaak ingezet bij **regressietesten**, waarbij wordt gecontroleerd of bestaande functionaliteiten nog correct werken na wijzigingen in de codebase. Dit is vooral van belang wanneer er geen volledige geautomatiseerde testset beschikbaar is.  
+
+Ook bij het **verifiëren van bugfixes** speelt handmatig testen een belangrijke rol. Nadat een defect is opgelost, wordt gecontroleerd of het probleem daadwerkelijk verholpen is. Dit gebeurt doorgaans als onderdeel van een hertest of retest. Daarnaast wordt **handmatige test**ing toegepast bij **usability testing**, waarbij de gebruikerservaring wordt beoordeeld. Hierbij kijkt men of de applicatie intuïtief en gebruiksvriendelijk is.
```
Comment (TeunvdKleij) [inline content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md:79]:
```suggestion
Ook bij het **verifiëren van bugfixes** speelt handmatig testen een belangrijke rol. Nadat een defect is opgelost, wordt gecontroleerd of het probleem daadwerkelijk verholpen is. Dit gebeurt doorgaans als onderdeel van een hertest of retest. Daarnaast wordt **handmatige testing** toegepast bij **usability testing**, waarbij de gebruikerservaring wordt beoordeeld. Hierbij kijkt men of de applicatie intuïtief en gebruiksvriendelijk is.  
```

File: content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md [hunk 25]
```diff
@@ -0,0 +1,87 @@
...
+
+Verschillende rollen binnen een team kunnen betrokken zijn bij **handmatige tests**. **Testers**, zoals **software test engineers** of **QA engineers**, voeren deze tests uit op basis van testmethodologieën en tools. **Ontwikkelaars** testen soms ook handmatig, vooral tijdens unit- en integratietests voordat zij code overdragen aan testers. Daarnaast kunnen business analisten of product owners **handmatige tests** uitvoeren om te verifiëren of de software voldoet aan zakelijke vereisten. Tot slot worden **handmatige test**s vaak uitgevoerd door eindgebruikers tijdens de User Acceptance Testing (UAT), waarbij de software in een realistische omgeving wordt getest om te bepalen of deze geschikt is voor productie. 
+
+---
+
+> Volgende stap: [[7. Uitleg evalueren testresultaat |Uitleg evalueren testresultaat]]
```
Comment (TeunvdKleij) [inline content/2. Processen/5. Testproces/6. Uitleg uitvoeren handmatige test.md:87]:
```suggestion
> Volgende stap: [[7. Uitleg evalueren testresultaat|Uitleg evalueren testresultaat]]
```

**Review by TeunvdKleij — CHANGES_REQUESTED** [review]

File: content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md
Comment (ernstbolt) [reply content/2. Processen/5. Testproces/4. Uitleg selecteren testsoorten.md:1]: Bestand heb ik teruggezet.

**Review by TeunvdKleij — APPROVED** [review]

**Review by martijnschuman — APPROVED** [review]
